### ISSUE TITLE
Feature Request: RedisClient::existsMany method

### ISSUE THREAD

**Issue by kagof** [vert-x3/vertx-redis-client#98]
### Version

* vert.x core: 3.5.0
* vert.x redis client: 3.5.0

### Context

For Redis commands which allow one or many keys, the RedisClient has methods which allow you to send multiple keys. Examples `RedisClient::mgetMany`, `RedisClient::scriptExistsMany`.

Since Redis version 3.0.3, the Redis [EXISTS](https://redis.io/commands/exists) command allows one or many keys, but the Vert.X RedisClient only has `RedisClient::exists`, which only allows one key. I would like to be able to send multiple keys in this command.

### Extra

* Redis documentation for this command is here: https://redis.io/commands/exists

**Comment by kagof** [comment]
I've created a pull request (#99) addressing this.

**Comment by pmlopes** [comment]
Fixed by #99

**Comment by vietj** [comment]
milestone should have been set to 3.5.1

**Comment by kagof** [comment]
I've just noticed that the test I wrote is failing on the build machine. I did some digging, and realized that the test suite creates a Redis server instance of version 2.8.19, and a Redis EXISTS call with multiple parameters requires Redis version ^3.0.3. It passed on my machine because I ran it against a local Redis server instance instead.

I've now run the test against the Redis instance that the tests create with some extra logging, and can confirm that this is the cause of the test failure - the result fails with the error message "ERR wrong number of arguments for 'exists' command".
